.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
We shouldn't have any IDE/OS specific files/directories here. Please revert and add it to your global `.gitignore`.

Reply:
resolved